ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -42,6 +42,8 @@ tests:
       CLUSTER_PROFILE: rosa-sts-ad
       QE_USAGE: rosa-sts-advanced-critical-high-f3
       REGION: ap-northeast-1
+      RHCS_CLUSTER_NAME_PREFIX: ci
+      RHCS_CLUSTER_NAME_SUFFIX: nightly
```

Comment:
It's not a nightly job. Why not put the frequency? like `f3`

Reply:
yeah. wanted generic but putting the frequency works as well.
I mostly wanted to identify which resources are for PR and which are for nightly.
Will update